docs/source/1.0/guides/converting-to-openapi.rst

```diff
@@ -691,6 +691,24 @@ disableCloudFormationSubstitution (``boolean``)
     .. seealso:: :ref:`openapi-cfn-substitutions`
 
 
+additionalAllowedCorsHeaders (``[string]``)
+    Sets additional allowed CORS headers on the preflight integration. If this
```

Comment:
I'd call this preflight requests and not integrations. We'll use this same configuration for HTTP APIs, and they don't require a preflight integration the same was as the REST APIs do.

Reply:
Updated the docs.